packages/btc/src/fungible.ts

```diff
@@ -0,0 +1,61 @@
+import { Config } from '@buildwithsygma/core';
+import type { networks } from 'bitcoinjs-lib';
+import type { BaseTransferParams, BitcoinTransferRequest, TypeOfAddress } from 'types';
+
+import { BaseTransfer } from './base-transfer.js';
+import { getPsbt } from './utils/index.js';
+
+export async function createBitcoinFungibleTransfer(
+  params: BaseTransferParams,
+): Promise<BitcoinTransfer> {
+  const config = new Config();
+  await config.init(process.env.SYGMA_ENV || params.environment);
+  return new BitcoinTransfer(params, config);
+}
+
+class BitcoinTransfer extends BaseTransfer {
+  protected destinationAddress: string;
+  protected amount: number;
+  protected publicKey: Buffer;
+  protected typeOfAddress: TypeOfAddress;
+  protected minerFee: number;
```

Comment:
remove this since we already provide feeRate and we have vsize to calculate the fee